src/components/LazyComponent/index.tsx

```diff
@@ -0,0 +1,77 @@
+import type {
+  PropsWithRef,
+  PropsWithChildren,
+  FunctionComponent,
+  ComponentType,
+} from 'react';
+import { Suspense, lazy } from 'react';
+
+import ErrorBoundary from 'components/ErrorBoundary';
+import LoadingOverlay from 'components/LoadingOverlay';
```

Comment:
Please, take notes of this: You should only add to the commit things that are used. Your commit needs to tell a story. `ErrorBoundary` should have been added at the same commit it's used here.

So, you're supposed to: 1. Create the LazyComponent with his use. 2. Create ErrorBoundary applying to LazyComponent. So in this way you do not create components that are not used like https://github.com/TirusTimes/tirus-times-front/pull/17/commits/f66c5772106540fe01b9557802784b7d1c1691bd